rotkehlchen/tests/api/test_ens.py

```diff
@@ -16,11 +18,17 @@
 from rotkehlchen.types import ChecksumEvmAddress, EnsMapping, Timestamp
 from rotkehlchen.utils.misc import ts_now
 
+if TYPE_CHECKING:
+    from rotkehlchen.api.server import APIServer
 
-def _get_timestamps(db: DBEns, addresses: list[ChecksumEvmAddress]):
```

Comment:
This makes more sense to have as addresses for type. So conversion should happen at the caller